internal/localworkflows/reportanalytics/migrations/0001_initial.sql

```diff
@@ -0,0 +1,10 @@
+-- +goose Up
+CREATE TABLE IF NOT EXISTS outbox (
+    id TEXT PRIMARY KEY,
+    timestamp INTEGER NOT NULL,
+    retries INTEGER DEFAULT 0,
+    payload BLOB NOT NULL
```

Comment:
If the payload is an opaque blob... how would a consumer of this outbox know whether it can process the contents? Is there a schema associated with it?